extension/intents/forms/forms.js

```diff
@@ -1,5 +1,18 @@
+/* globals buildSettings */
+
 import * as intentRunner from "../../background/intentRunner.js";
 import * as content from "../../background/content.js";
+import * as pageMetadata from "../../background/pageMetadata.js";
+import * as searching from "../../searching.js";
+
+function createTabGoogleLucky(query, options = {}) {
```

Comment:
You should add the hide option to the `createTabGoogleLuck` method in `intentRunner.js`. That function does some other important things you'll want to use for this as well.